### DISCUSSION TITLE
How about n8n support ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#14175] [Wishlist]
Hi,

[n8n](https://n8n.io/) is free and open fair-code licensed node based Workflow Automation Tool.
It could be nice to see Yetiforce support this in order to connect with other apps.
https://docs.n8n.io/nodes/creating-nodes/create-node.html
Around 200 other apps like Dropbox, Google calendar, Trello, Mailchimp, Wordpress, Zendesk, etc. are already integrated.
https://n8n.io/integrations

What do you think ?

**Comment by mariuszkrzaczkowski** [comment]
nice solution, have already seen some similar
the question of who will pay for it is not cheap / quick to do
